### ISSUE TITLE
How to collapse node at initialization？

### ISSUE THREAD

**Issue by lynings** [valor-software/ng2-tree#102]
And how to set whether node is expanded？
Because In the initialization or call loadChildren function, when the tree is too high, this is a very bad user experience.

**Comment by rychkog** [comment]
@lynings don't have this option - one possibility to do it - by using API developed in scope of https://github.com/valor-software/ng2-tree/pull/78

The other, with an ability to define in settings which node should be collapsed, is being developed.

**Comment by rajeshkdev** [comment]
"The other, with an ability to define in settings which node should be collapsed, is being developed." 

@rychkog Any updated on the above comment of yours? 
Is it now possible to define in settings that node should be collapsed/expanded?

**Comment by rychkog** [comment]
@rajeshkdev Will be available in a next release (during the next week).

**Comment by rajeshkdev** [comment]
@rychkog Thank you so much. That would solve my big headache. Looking forward your updates.

**Comment by rajeshkdev** [comment]
This implementation is available ng2-tree v2.0.0-rc.2 version onwards.

The following flag need to be set to collapse by default. Else it will be all the expanded .

> isCollapsedOnInit: true,


```
public treeViewConfig() {
        const ffs: TreeModel = {
            value: this.appConfig.rootNode,
            id: 1,
            settings: {
                isCollapsedOnInit: true,
                cssClasses: {
                    expanded: 'fa fa-caret-down',
                    collapsed: 'fa fa-caret-right',
                    empty: 'fa fa-caret-right disabled',
                    leaf: 'fa'
                },
                templates: {
                    node: '<i class="fa fa-folder"></i>',
                    leaf: '<i class="fa fa-file"></i>'
                },
                leftMenu: false,
                rightMenu: false
            },
            children: []
        };
        return ffs;
    }
```
